gcp_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# 
+# Prepare GCP project environment by creating the required bucket and dataset.
+
+#######################################################
+# Run command and check return status, exit if fails.
+# Arguments:
+#   $1: command to be run
+# Returns:
+#   None
+#######################################################
+exit_if_failed_on_command() {
+  echo "executing "$1
+  $1
+  if [[ $? -eq 0 ]]; then
+    echo "successful"
+  else
+    exit 1
+  fi
+}
+
+echo "Preparing GCP environment..."
+commands_to_run=( "gsutil mb gs://airflow-gcp-smoke" "bq mk airflow"
```

Comment:
You don't understand. airflow-gcp-smoke is globally scoped. This means if person x creates airflow-gcp-smoke, nobody in the world can create the same bucket.

Actually it's a bad idea of doing the setup with a script that does automaticly create buckets and datasets. We use 10's of projects and have our data distributed over them. 

That's why the variables exist in Airflow and the manual setup. I'm going to reject the script.

If you remove the script and squash you commit I will accept the README change but you will have to be fast because I'll be syncing the examples with a blog series that I will start publishing.